cmd/buildah/run.go

```diff
@@ -14,18 +14,17 @@ import (
 )
 
 type runInputOptions struct {
-	addHistory     bool
-	capAdd         []string
-	capDrop        []string
-	hostname       string
-	isolation      string
-	runtime        string
-	runtimeFlag    []string
-	noPivot        bool
-	securityOption []string
-	terminal       bool
-	volumes        []string
-	mounts         []string
+	addHistory  bool
+	capAdd      []string
+	capDrop     []string
+	hostname    string
+	isolation   string
+	runtime     string
+	runtimeFlag []string
+	noPivot     bool
+	terminal    bool
+	volumes     []string
+	mounts      []string
```

Comment:
I know it's not yours, but everything else in this list is alpha order, could you move `mounts` up if you have other changes?

Reply:
nice catch, thanks. Fixed; push pending resolution of the `--userns` question.